### PR TITLE
Added named volume parameters and explanation

### PR DESCRIPTION
**Pull request by vitalykorolev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,32 +43,24 @@ $ docker run -d -it -p 8000:8000 -p 8001:8001 -p 8002:8002 \
...
-
-Verify volume creation with this command:
+The following command will list previously created volumes:
 
 ```
 $ docker volume ls
 ```
 
-Alternately, you can bind a volume at runtime using the `-v` option:
+The command should output at least two randomly generated volume identifiers from the previous commands.
```
Comment (twosdai) [inline README.md:56]: I think we should add the following output  as an example snippet afterwards: 

Example output: 
```
DRIVER    VOLUME NAME
local     0f111f7336a5dd1f63fbd7dc07740bba8df684d70fdbcd748899091307c85019
local     1b65575a84be319222a4ff9ba9eecdff06ffb3143edbd03720f4b808be0e6d18
```

File: README.md [hunk 2]
```diff
@@ -43,32 +43,24 @@ $ docker run -d -it -p 8000:8000 -p 8001:8001 -p 8002:8002 \
...
+
+The following command uses named volume in order to make it easier to manage:
```
Comment (twosdai) [inline README.md:58]: I think the following reads a little bit better but let me know what you think: 

`The following command uses a named volume in order to make management easier`

File: README.md [hunk 3]
```diff
@@ -96,7 +88,7 @@ MarkLogic Docker containers are configured via a set of environment variables.
 
 MarkLogic Docker containers ship with a small set of scripts, making it easy to create clusters. Below are three examples for creating MarkLogic clusters with Docker containers. The first two use [Docker compose](https://docs.docker.com/compose/) scripts to create one-node and three-node clusters. The third example demonstrates a container setup on separate VMs.
 
-The credentials for admin user are configured via Docker secrets, and are stored in mldb_admin_username.txt and mldb_admin_password.txt files. Make sure the yaml files for Docker compose have the desired image label and volume path (~/data/MarkLogic). All of these scripts were tested with version 20.10 of Docker.
+The credentials for admin user are configured via Docker secrets, and are stored in mldb_admin_username.txt and mldb_admin_password.txt files.
```
Comment (twosdai) [inline README.md:91]: The credentials for `the` admin user

File: README.md [hunk 4]
```diff
@@ -120,7 +112,7 @@ services:
         - MARKLOGIC_ADMIN_PASSWORD_FILE=mldb_admin_password
         - TZ=Europe/Prague
       volumes:
-        - ~/data/MarkLogic:/var/opt/MarkLogic
+        - MarkLogic:/var/opt/MarkLogic
```
Comment (twosdai) [inline README.md:115]: Do you think our naming convention for these yaml files should include a prefix or suffix containing what type of cluster it is? 

I can forsee clients wanted to run through the whole readme and having a few Volumes on their machine from the single node and three node example. 

Maybe something like 
`Marklogic_sn:/var/opt/MarkLogic`
for here and 
`MarkLogic1_mn:/var/opt/MarkLogic`  for below in the 3-node example. 

No big deal either way just a thought.

Comment (twosdai) [reply README.md:115]: If we do end up going with it, we would also want to change the docker compose example.

File: README.md (hunk 1 above)
Comment (twosdai) [reply README.md:56]: Furthermore do we want to add an example output after the named volumes have been created?

File: README.md (hunk 4 above)
Comment (vitalykorolev) [reply README.md:115]: Agreed. I wanted to bring this up as well after we merged into release branch.

**Comment by twosdai** [comment]
LGTM

**Pull request merged**